realm/realm-library/src/androidTest/java/io/realm/OrderedRealmCollectionIteratorTests.java

```diff
@@ -801,6 +800,7 @@ public void run() {
     }
 
     @Test
+    @Ignore("Enable this test when support RealmCollectionSnapshot")
```

Comment:
It is, see https://github.com/realm/realm-java/pull/4172/files#diff-07a5d547f084d754d3b6e6b0fdfeff26R834